compiler/luci/import/src/GraphBuilderRegistry.cpp

```diff
@@ -30,6 +30,7 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   CIRCLE_NODE(ABS, CircleAbsGraphBuilder);                                                 // 101
   CIRCLE_NODE(ADD, CircleAddGraphBuilder);                                                 // 0
   CIRCLE_NODE(ARG_MAX, CircleArgMaxGraphBuilder);                                          // 56
+  CIRCLE_NODE(ARG_MIN, CircleArgMinGraphBuilder);                                          // 79
```

Comment:
I think you need to remove `// BuiltinOperator_ARG_MIN = 79,` line

Reply:
Oh! Sorry I will fix it!